src/App.tsx

```diff
@@ -1,39 +1,65 @@
-import React from 'react';
+import { Component } from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+interface State {
+  hasClock: boolean,
```

Comment:
```suggestion
  isClockVisible: boolean,
```

Reply:
It makes sense, but the name `hasClock` was [predefined in the task description](https://prnt.sc/CMOmD-HJNGa3).

Reply:
you're right 👍 I didn't look at the description, as for me it's not clear naming 😎 , thanks for the answer